modules/tracing/mule-tracer-exporter-impl/src/main/java/org/mule/runtime/tracer/exporter/impl/OpenTelemetrySpanExporterFactory.java

```diff
@@ -91,6 +94,7 @@ public SpanExporter getSpanExporter(InternalSpan internalSpan, InitialSpanInfo i
 
   protected SpanProcessor resolveOpenTelemetrySpanProcessor() {
     if (isExportEnabled()) {
+      LOGGER.info("{} is enabled.", MULE_OPEN_TELEMETRY_EXPORTER_ENABLED);
```

Comment:
Maybe we could log a message instead of the name of the property for this case, like "Mule OpenTelemetry Exporter is enabled"